app/helpers/blacklight_helper.rb

```diff
@@ -269,43 +271,19 @@ def truncated_link(doc, field, opts = { counter: nil }, length = 200)
 
     ##
     # Check if any search parameters have been set
-    # @return [Boolean]
-    def search_parameters?
-      blacklight_params[:q].present? || blacklight_params[:f].present? || blacklight_params[:search_field].present?
-    end
-
-    # Determines whether or not a query is provided in the request
-    # @param [ActionController::Parameters] params
-    # @return [Boolean]
-    def empty_search_query?(params)
-      params[:q].blank?
-    end
-
-    # Determines whether or not the request is scoped for all fields
-    # @param [ActionController::Parameters] params
-    # @return [Boolean]
-    def all_search_fields_query?(params)
-      params[:search_field] && params[:search_field] == 'all_fields'
-    end
-
-    # Determines whether or not the request is scoped for a specific page number
-    # @param [ActionController::Parameters] params
-    # @return [Boolean]
-    def first_page_query?(params)
-      !params[:page]
-    end
-
-    # Determines whether or not the request facets upon any fields
     # @param [ActionController::Parameters] params
     # @return [Boolean]
-    def faceted_query?(params)
-      params[:f]
+    def search_parameters?(params)
+      params[:q].present? || params[:f].present?
```

Comment:
Change `params[:q].present?` to `params[:q].nil?`
With the current behavior clicking a blank search and then paging to the second one errors. I think we should allow this behavior.

Reply:
👍 but I'm going to create an issue for whether we should allow paging without a query or facet being selected.  I would really like to know why people are doing this and think about other ways to help them find what they're looking for.